doc/source/user_guide/solver_settings.rst

```diff
@@ -10,10 +10,17 @@ settings using :ref:`ref_solver_tui_commands`:
 
 Selecting steady or transient
 -----------------------------
-The ``define.models.steady`` TUI enables and disables the steady solution model.
+The following example shows a comparison between the TUI command and the
+python code for enabling and disabling the steady and unsteady solution model.
 
-The ``define.models.unsteady.1st_order TUI`` selects the first-order implicit
-formulation for a transient simulation.
+TUI command
+
+.. code:: scheme
+
+    /define/models/steady yes
+    /define/models/unsteady_1st_order yes
+
+Python command
 
 .. code:: python
```

Comment:
The python code launches fluent and reads a case, maybe we can leave that out.  Probably it's ok to assume in the examples that the someone has launched a session as described in the "Launching Fluent locally" section:

https://github.com/pyansys/pyfluent/blob/main/doc/source/user_guide/launching_ansys_fluent.rst